src/persist/src/indexed/runtime.rs

```diff
@@ -37,25 +37,13 @@ use crate::indexed::cache::BlobCache;
 use crate::indexed::columnar::{ColumnarRecords, ColumnarRecordsBuilder};
 use crate::indexed::encoding::Id;
 use crate::indexed::metrics::Metrics;
-use crate::indexed::{Indexed, ListenFn, Snapshot};
+use crate::indexed::{Cmd, Indexed, ListenFn, Snapshot};
 use crate::pfuture::{PFuture, PFutureHandle};
 use crate::storage::{Blob, Log, SeqNo};
 use futures_executor::block_on;
 
-#[derive(Debug)]
-enum Cmd {
-    Register(String, (String, String), PFutureHandle<Id>),
-    Destroy(String, PFutureHandle<bool>),
-    Write(Vec<(Id, ColumnarRecords)>, PFutureHandle<SeqNo>),
-    Seal(Vec<Id>, u64, PFutureHandle<SeqNo>),
-    GetDescription(String, PFutureHandle<Description<u64>>),
-    AllowCompaction(Vec<(Id, Antichain<u64>)>, PFutureHandle<SeqNo>),
-    Snapshot(Id, PFutureHandle<ArrangementSnapshot>),
-    Listen(
-        Id,
-        ListenFn<Vec<u8>, Vec<u8>>,
-        PFutureHandle<ArrangementSnapshot>,
-    ),
+enum RuntimeCmd {
+    Cmd(Cmd),
```

Comment:
I maybe have a mild preference to calling this variant `IndexedCmd`?

Reply:
yeah that wfm